tests/dat/actions/hello.py

```diff
@@ -19,12 +19,9 @@
 """
 
 
-def main(dict):
+def main(args_dict):
```

Comment:
In Swift actions the parameter name is "`args`" and I think Node actions use "`params`". Maybe pick one of those two?